tests/test_examples.py

```diff
@@ -212,3 +214,89 @@ def enable_brake(*args, **kwargs):
     mocker.patch.object(MotionController, "configuration", MockConfiguration)
     result = script_runner.run(script_path, override, dictionary, f"-ip={ip_address}")
     assert result.returncode == 0
+
+
+@pytest.mark.virtual
+def test_can_bootloader_example_success(mocker, capsys):
+    device = "my_can_device"
+    channel = 111
+    baudrate = 4444
```

Comment:
These values are weird and in some cases make no sense. I would use normal ones.